tests/Tester.py

```diff
@@ -48,6 +51,28 @@ def __init__(self, source_range: SourceRange, exception_type: type(Exception)):
         self.exception_type = exception_type
 
 
+@cache
+def get_hedy_source_hash():
+    directory = os.path.join(ROOT_DIR, 'grammars')
+
+    # should be kept in sync with .github/workflows/generate-snippet-hashes.yml
```

Comment:
This seems a bit dangerous. Why do we have the hash in both again? 

Can we not have it in one place?

Reply:
This comment needs to go as we remove that workflow in this PR. Sorry about that.

Reply:
Removed.